docs/source/documentation.rst

```diff
@@ -182,9 +182,9 @@ Existing models
 Three models are available and shipped together with the installation package, so you don't need to install them separately.
 The three models are described below:
 
-* A SEM model, that works at a resolution of 0.1 micrometer per pixel.
-* A TEM model, that works at a resolution of 0.01 micrometer per pixel.
-* A BF (bright-field) model, that works at a resolution of 0.1 micrometer per pixel.
+* A SEM model, that works at a resolution of 0.1 micrometer per pixel. For more information, please visit the `SEM model repository <https://github.com/axondeepseg/default-SEM-model>`_.
```

Comment:
Maybe we could add the name of the model itself, ex:
```suggestion
* A SEM model (model_seg_rat_axon-myelin_sem), that works at a resolution of 0.1 micrometer per pixel. For more information, please visit the `SEM model repository <https://github.com/axondeepseg/default-SEM-model>`_.
```

Reply:
Good suggestion this is a good opportunity to document their names. We had a question about these names recently.

Reply:
I just remembered, we had an issue about this as well #573. It might be a good time to think about how to track "old" model names when we changed them, as discussed in the issue.

Reply:
Hmm that's interesting. I can add it manually easily but I don't know how we can automate this with RTD. Are the model names likely to be changed again? 
The problem is that I don't even know where we would fetch this information. For example, at this point, where is the `model_seg_pns_bf` name referenced? The "crawler" would need to look into old commits and check for hardcoded model names, but it sounds overkill to me. For example, this command:
```bash
git log --pretty=short -u -L 12,14:AxonDeepSeg/download_model.py
```
tells us that `default_SEM_model`, `default_TEM_model` and `model_seg_pns_bf` were resp. changed for `model_seg_rat_axon-myelin_sem`, `model_seg_mouse_axon-myelin_tem` and `model_seg_rat_axon-myelin_bf`. However, I don't think this is very robust.

Reply:
Maybe we could add a note in https://github.com/axondeepseg/axondeepseg/wiki/Instructions-on-How-to-Update-Models-in-Repo-and-Github-Storage that the doc needs to be updated after a model name change but will the next person that updates a model read this wiki?

Reply:
Hi @hermancollin, sorry for the delay.
As a first step, I think we should add the mention "formely called ..." for our models names in the documentation.
In fsleyes for example, the displayed name changed between v3 and v4 and can create confusion for the user following the video tutorial (ex: in this [comment](https://github.com/axondeepseg/axondeepseg/discussions/614#discussion-3940183)).
In addition, we are publishing a paper with the "old name" of the BF model and it would be best to document it (if I recall correctly, this is what triggered the issue #573 in the first place).

Reply:
I added the BF model "legacy" name.